src/clients/HubPoolClient.ts

```diff
@@ -175,7 +175,7 @@ export class HubPoolClient extends BaseAbstractClient {
     return l1Token;
   }
 
-  getL1TokenCounterpartAtBlock(l2ChainId: number, l2Token: string, block: number): string {
+  getL1TokenCounterpartAtBlock(l2ChainId: number, l2Token: string, l1Block: number): string {
```

Comment:
Thoughts on `hubPoolBlock` instead of `l1Block` ?

We will soon support SpokePool deployments to other L1s, so the existing naming conventions of `l1`and `l2` terms won't make as much sense and IMO should be phased out.

Reply:
yes good point!